src/HasMedia/HasMediaTrait.php

```diff
@@ -69,6 +69,28 @@ public function addMediaFromRequest(string $key)
     }
 
     /**
+     * Add multiple files from a request by keys.
+     *
+     * @param string[] $keys
+     *
+     * @return \Spatie\MediaLibrary\FileAdder\FileAdder[]
+     */
+    public function addMultipleMediaFromRequest(array $keys)
```

Comment:
Didn't notice the return type was different, so yeah makes sense to have a different method. And it would indeed be a breaking change since we'd have to change the signature.